tests/python/test_impactx.py

```diff
@@ -81,6 +81,11 @@ def test_impactx_nofile():
     """
     sim = ImpactX()
 
+    # various ways to change OMP threads from the default "nosmt"
+    sim.omp_threads = 1
+    sim.omp_threads = "1"
+    sim.omp_threads = "nosmt"
```

Comment:
The string "nosmt" seems opaque to me.  Is there a more descriptive string that would work better for the default here?